obspy/io/sac/core.py

```diff
@@ -36,7 +36,11 @@ def _is_sac(filename):
 
     .. rubric:: Example
 
-    >>> _is_sac('/path/to/test.sac')  #doctest: +SKIP
```

Comment:
@jkmacc-LANL FYI `'/path/to'` magic is resolved by the `@map_example_filename` decorator (and we only want that magic on the top-level namespace helpers like `read()` so it's correct that `_is_sac` doesn't have that magic).